firefox-ios/Client.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -140,8 +140,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/apple/swift-crypto.git",
       "state" : {
-        "revision" : "21f7878f2b39d46fd8ba2b06459ccb431cdf876c",
-        "version" : "3.8.1"
+        "revision" : "06dc63c6d8da54ee11ceb268cde1fa68161afc96",
+        "version" : "3.9.1"
```

Comment:
remove?

Reply:
ok thanks, let me know when this PR is ready to review! I also had some comments related in the previous PR as well with the changes that also exist here.
https://github.com/mozilla-mobile/firefox-ios/pull/22899

Reply:
@cyndichin  I've removed the edits, should be ready for review now